docs/how-to/ccv0.sh

```diff
@@ -199,47 +210,38 @@ build_and_install_kata_runtime() {
 }
 
 configure() {
+    local containerd_config="/etc/containerd/config.toml"
+
     debug_function configure_kata_to_use_rootfs
     debug_function enable_full_debug
     # Temp PoC verify code: Inject policy path config parameter
-    sudo sed -i -e 's%^kernel_params = "\(.*\)"%kernel_params = "\1 agent.container_policy_file=/etc/containers/quay_verification/quay_policy.json"%g' /etc/kata-containers/configuration.toml
-
+    add_kernel_params
+    
     # If using AA then need to add the agent_config
     if [ "${AA_KBC}" == "offline_fs_kbc" ]; then
-        sudo sed -i -e 's%^kernel_params = "\(.*\)"%kernel_params = "\1 agent.config_file=/etc/agent-config.toml"%g' /etc/kata-containers/configuration.toml
+        add_kernel_params
```

Comment:
Are these `add_kernel_params()` calls missing an arg?

Reply:
ah yep good spot - I completely missed that... I've updated now